icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/MeasureUnitTest.java

```diff
@@ -85,7 +85,7 @@ public int compareTo(OrderedPair<F, S> other) {
         }
     }
 
-    private static final String[] DRAFT_VERSIONS = {"68", "69"};
+    private static final String[] DRAFT_VERSIONS = {"70", "71"};
```

Comment:
```suggestion
    private static final String[] DRAFT_VERSIONS = {"69", "70"};
```
right?

Reply:
No, the versions we want to be in draft status are 70 and 71; 69 and earlier should be promoted to stable in this release.

Reply:
Sorry, yes, you are right. I just saw it jumping farther than it normally should, but yes it should have been incremented last time.